.github/workflows/ts.yml

```diff
@@ -1,3 +1,5 @@
+name: Lookout
```

Comment:
Should we make this name be more descriptive?  This is really a javascript unit testing and linting.  Lookout has a backend Go component and a UI.  This action tests javascript only.

Reply:
Tell me what to call it and I'll call it that :)

Reply:
After I posted this I looked at what they are named in the actions

Lookout / ts-lint is fine